### ISSUE TITLE
Error when trying to install with Helm.

### ISSUE THREAD

**Issue by jonathanhatch** [storageos/charts#102]
helm install storageos/storageos-operator --namespace storageos-operator --generate-name
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
manifest_sorter.go:192: info: skipping unknown hook: "crd-install"
Error: unable to build kubernetes objects from release manifest: unable to recognize "": no matches for kind "StorageOSCluster" in version "storageos.com/v1"

**Comment by muhlba91** [comment]
are you using Helm 3? if so, #100 should address this issue.

**Comment by avestuk** [comment]
@jonathanhatch Currently helm3 is not supported and as @muhlba91 pointed out #100 will address this. Unfortunately as helm3 charts are not backwards compatible with helm2 we are still discussing how to handle this. 

If you have a hard requirement for helm3 then can I suggest you try a manual installation using the StorageOS operator instead? https://docs.storageos.com/docs/install/kubernetes/

**Comment by jonathanhatch** [comment]
Yeah, that kind of defeats the purpose of using Helm as a package manager. It's problematic for me because the manual installation of StorageOS doesn't work. I just stood up a containerd cluster with two nodes to test manual installation and the final CRD trigger isn't triggering any kind of installation.

**Comment by avestuk** [comment]
A two node installation will not work if you are using embedded etcd as etcd cannot establish quorum between two nodes. Can you try and use a node selector to limit the installation to a single node, add a third node, or use [external etcd](https://docs.storageos.com/docs/operations/external-etcd/)? 

It may also be helpful if you can show the output of a `kubectl describe StorageOSCluster`
